tests/mlmodel_openai/test_chat_completion.py

```diff
@@ -41,94 +42,101 @@ def set_trace_info():
     {"role": "user", "content": "What is 212 degrees Fahrenheit converted to Celsius?"},
 )
 
+_test_openai_chat_completion_async_messages = (
```

Comment:
This was changed in the sync PR but we realistically don't need all the messages in this list to be tested since we have multiple user and assistant response coming out of this. Given that there isn't a benefit to validating additional message events, this can match what is in the sync PR. That includes just the first two messages.